starport/cmd/network_chain_prepare.go

```diff
@@ -57,5 +59,6 @@ func networkChainPrepareHandler(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	fmt.Println("🛠️  After launch triggered, start your node by running: `{binary} --home {home} start`")
```

Comment:
you should put after the chain prepare command, and also you must dynamically set the binary name into `{binary}` and the home folder into the `{home}`

Reply:
We also need to write the actual binary name and home path for the chain instead of placeholders inside curly brackets.